website/source/docs/upgrade/upgrade-specific.html.md

```diff
@@ -25,8 +25,24 @@ images to load, in 0.5.5 it has been changed to a single string. No
 functionality was changed. Even if more than one item was specificed prior to
 0.5.5 only the first item was used.
 
-If you have jobs that use the `load` option first upgrade all nodes to 0.5.5,
-then resubmit those jobs with `load` as a single string.
+To do a zero-downtime deploy with jobs that use the `load` option:
+
+* Upgrade to servers to 0.5.5 or later.
```

Comment:
"Upgrade to servers to 0.5.5 or later." -> "Upgrade servers to version 0.5.5 or later."